bears/python/PycodestyleBear.py

```diff
@@ -0,0 +1,48 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='pycodestyle',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+) (?P<column>\d+) (?P<message>.*)')
```

Comment:
if you want, you can put the code into the `origin` capture group, the result will then appear as `PycodestyleBear (E123)`